README.md

```diff
@@ -43,6 +43,19 @@ Or using tox command:
 ```
 $ tox -e venv -- controller
 ```
+### Run Unit Tests
+
+In order to execute a unit test of a specific class run the following command:
+
+```
+$ pytest controller/test/unit/path/to/test/test_class.py
+```
+
+Or run all test cases using tox command:
+
+```
+$ tox
```

Comment:
nit: as we're not currently fully supporting Python 3 (as opposed to what says the README), I would specify the py27 environment here for tests: tox -epy27.

Reply:
Agreed.